### ISSUE TITLE
I wonder why do log_softmax twice on the prediction logits?

### ISSUE THREAD

**Issue by lixinsu** [kevinduh/san_mrc#16]
As depicted in code bellow . 
https://github.com/kevinduh/san_mrc/blob/1deb1b3e9e1f2399f3a16f3bf4be9db44f5d10f6/src/san.py#L105 . 
https://github.com/kevinduh/san_mrc/blob/1deb1b3e9e1f2399f3a16f3bf4be9db44f5d10f6/src/model.py#L92 .

**Comment by lixinsu** [comment]
Is this a special procedure for SAN module?

**Comment by namisan** [comment]
It is only to make the api consistence, which is output of logits.
